File: cms-adapter/src/Rsbc.Dmf.CaseManagement/Model/Dto/Document.cs
using System;

namespace Rsbc.Dmf.CaseManagement.Dto
{
    // Dynamics schema name bcgov_documenturl
    public class Document
    {
        // dfp_dmertype
        public string DmerType { get; set; }

        // dfp_dmerstatus
        public string DmerStatus { get; set; }

        // dfp_compliancedate
        public DateTimeOffset? ComplianceDate { get; set; }

        // bcgov_caseid
        public Case Case { get; set; }

        public DateTimeOffset CreatedOn { get; set; }
      
        public string Description { get; set; }

        public string SubmittalStatus { get; set; }
        public DocumentType DocumentType { get; set; }

        public DocumentSubType DocumentSubType { get; set; }

        public string DocumentUrl { get; set; }
        public Login Login { get; set; }
    }
}

File: medical-practitioner-portal/src/app/shared/api/models/document.ts
/* tslint:disable */
/* eslint-disable */
import { Case } from '../models/case';
import { DocumentSubType } from '../models/document-sub-type';
import { DocumentType } from '../models/document-type';
import { Timestamp } from '../models/timestamp';
export interface Document {
  case?: Case;
  complianceDate?: Timestamp;
  createdOn?: Timestamp;
  description?: string | null;
  dmerStatus?: number;
  dmerType?: number;
  documentSubType?: DocumentSubType;
  documentType?: DocumentType;
  documentUrl?: string | null;
  submittalStatus?: string | null;
}
